projects/kit/components/multi-select/multi-select.component.ts

```diff
@@ -254,8 +265,15 @@ export class TuiMultiSelectComponent<T>
         this.updateValue(value.map(({item}) => item));
     }
 
+    onValueChange(value: readonly T[]): void {
+        this.updateValue(value);
+    }
+
     onSearch(search: string | null): void {
-        this.open = true;
+        if (!this.nativeDropdownMode) {
```

Comment:
When does `onSearch` trigger in native dropdown mode and why does it matter not to change `open`? Don't we already have `canOpen` that should not allow it to open?

Reply:
I noticed a bug when clicking on cleaner: dropdown doesn't open but arrow icon rotates. But now I can't reproduce it 🤔  deleted